.gitmodules

```diff
@@ -5,4 +5,6 @@
 [submodule "regression-tests"]
 	path = regression-tests
 	url = https://github.com/pulp-platform/regression_tests.git
-    branch = astral
+[submodule "fault_injection_sim"]
+	path = fault_injection_sim
+	url = git@iis-git.ee.ethz.ch:michaero/fault_injection_sim.git
```

Comment:
Use either `https` or `ssh` everywhere for consistency!

Reply:
(`https` is preferrable)

Reply:
This was needed as the repository was originally hosted internally to GitLab. Since it is now freely available on the public GitHub, a later commit dropped this reference from the `.gitmodules`. See https://github.com/pulp-platform/pulp_cluster/commit/5680405054a4ed3d346dc8591300156b7a25d882.